package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "prepare": "node prepare-swagger-ui",
-    "test": "standard && tap --100 test/*.js && npm run typescript",
+    "test": "standard && tap --no-esm --100 test/*.js test/esm/*.js && npm run typescript",
```

Comment:
https://github.com/fastify/fastify/blob/master/.taprc < use this instead.

Reply:
Done